apps/win32/package.json

```diff
@@ -27,11 +27,12 @@
     "generate-report": "allure generate allure-results --clean && allure open"
   },
   "dependencies": {
-    "@fluentui-react-native/tester": "^0.4.1",
+    "@fluentui-react-native/tester": "^0.5.1",
     "@uifabricshared/theming-react-native": "0.7.76",
     "react": "16.11.0",
     "react-native": "0.62.2",
-    "react-native-svg": "^11.0.0"
+    "react-native-svg": "^11.0.0",
+    "react-native-windows": "^0.62.1"
```

Comment:
Why the rnw dependency? This project uses @office-iss/react-native-win32

Reply:
Without this line one of the CI checks fail:

ERR! ERROR DETECTED IN @fluentui-react-native/tester-win32 depcheck
ERR! started
ERR! hash: 08d7269c5dfeb123def1002650ff1e81b1ccf88d, cache hit? false
ERR! Running /tmp/yarn--1599700209945-0.3433538225809185/yarn run depcheck
ERR! warning package.json: No license field
ERR! $ fluentui-scripts depcheck
ERR! [1:20:42 AM] ■ started 'depcheck'
ERR! [1:20:42 AM] ▲ Unused devDependencies
ERR! [1:20:42 AM] ▲ -- @react-native-community/eslint-config
ERR! [1:20:42 AM] ▲ -- @types/jasmine
ERR! [1:20:42 AM] ▲ -- metro-react-native-babel-preset
ERR! [1:20:42 AM] ▲ -- tsconfig-paths
ERR! [1:20:42 AM] ▲ -- allure-commandline
ERR! [1:20:42 AM] ▲ -- appium
ERR! [1:20:42 AM] ▲ -- webdriverio
ERR! [1:20:42 AM] ▲ -- @wdio/allure-reporter
ERR! [1:20:42 AM] ▲ -- @wdio/appium-service
ERR! [1:20:42 AM] ▲ -- @wdio/cli
ERR! [1:20:42 AM] ▲ -- @wdio/jasmine-framework
ERR! [1:20:42 AM] ▲ -- @wdio/local-runner
ERR! [1:20:42 AM] ▲ -- @wdio/spec-reporter
ERR! [1:20:42 AM] ▲ -- @wdio/sync
ERR! [1:20:42 AM] x Missing dependency on react-native-windows

Reply:
The line below in the log is actually the relevant one. It's coming from the usage in react-native.config.js having a reference to react-native-windows. The unused devDependencies are just warnings.

Reply:
If that file goes away or references react-native-win32 instead the depcheck error will go away. You can also run yarn depcheck at the root to test it before CI, though I would recommend running yarn depcheck --no-cache just to be sure.

Reply:
Changed the refence to react-native-win32 and it looks ok now.